client/rest/src/test/java/org/elasticsearch/client/RequestOptionsTests.java

```diff
@@ -133,6 +137,9 @@ private static RequestOptions mutate(RequestOptions options) {
             mutant.addHeader("extra", "m");
             return mutant.build();
         case 1:
+            mutant.setNodeSelector(mock(NodeSelector.class));
+            return mutant.build();
+        case 2:
```

Comment:
I am afraid this one will never run ;)

Reply:
:sob: If we had Java 9 and our test framework I'd say we should use `randomFrom` and pick a `Consumer<RequestOptions.Builder>` from a list then this kind of thing can't happen.